src/state/shell/composer/index.tsx

```diff
@@ -21,7 +20,7 @@ export interface ComposerOptsPostRef {
   cid: string
   text: string
   author: AppBskyActorDefs.ProfileViewBasic
-  embed?: AppBskyEmbedRecord.ViewRecord['embed']
+  embed?: AppBskyFeedDefs.PostView['embed']
```

Comment:
This type was just entirely wrong. `embed` does not exist on `AppBskyEmbedRecord.ViewRecord`. All the usages, however, were correct 😅